test/Table_Tests/src/Common_Table_Operations/Select_Columns_Spec.enso

```diff
@@ -229,20 +229,42 @@ add_specs suite_builder setup =
             data.teardown
 
         group_builder.specify "should be able to select by type of columns" <|
+            expect_column_names ["int"] <| data.table.select_columns [..By_Type ..Integer]
```

Comment:
Wow, seeing the doubly nested auto-scoping looks really interesting, great that it works 😁